src/Interpreters/AsynchronousMetrics.cpp

```diff
@@ -208,7 +208,6 @@ void AsynchronousMetrics::update()
         /// Otherwise it might be calculated incorrectly - it can include a "drift" of memory amount.
         /// See https://github.com/ClickHouse/ClickHouse/issues/10293
         total_memory_tracker.set(data.resident);
-        CurrentMetrics::set(CurrentMetrics::MemoryTracking, data.resident);
```

Comment:
Is it intended?

Reply:
Yes, it will be adjusted during limit check in `MemoryTracker::alloc`

Reply:
Ok, but the value will still contain drift.

Reply:
But w/o this ClickHouseMetrics_MemoryTracking can grow over rss (since in MemoryTracker::alloc it is reseted only if the the limit reached), I will get revert this hunk

https://github.com/ClickHouse/ClickHouse/pull/10362#issuecomment-618079793